packages/react-core/src/components/Banner/Banner.tsx

```diff
@@ -11,12 +11,15 @@ export interface BannerProps extends React.HTMLProps<HTMLDivElement> {
   variant?: 'default' | 'info' | 'danger' | 'success' | 'warning';
   /** If set to true, the banner sticks to the top of its container */
   isSticky?: boolean;
+  /** Text announced by screen reader when variant is set to indicate the type of banner*/
+  screenReaderText?: string;
```

Comment:
Thinking some slight tweaking of the prop description could be done here, since the variant will always be set and to mention what the default is:

"Text announced by screen readers to indicate the type of banner. Defaults to `${variant} banner` if this prop is not passed in.".

Reply:
Thanks!
Now it's more clear 👍🏼